### PR TITLE
Removed ProteinSimulation from main README

### PR DESCRIPTION
**Pull request by david451lima**
REmoved proteinSolvation part of the main readme. Now we have the documentation in the scenario folder.
Also now plot rmsf returns rmsf to be consistent with the documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,43 +40,6 @@ And you are good to go! You can start [exploring our tutorial notebooks](https:/
...
-### Protein solvation
-
-The protein solvation scenario models the dynamics of a protein whose structure is described by a PDB file. The protein is placed in a cubic box filled with water. If the protein has a non-zero electric charge, charged ions are added to the solution to neutralize the system. First, the system undergoes an [energy minimization](https://manual.gromacs.org/current/reference-manual/algorithms/energy-minimization.html) process to eliminate any steric clashes or structural issues within the protein-water system. After this, the position of the atoms in this system is updated according to Newton's equation in discrete time steps. The force that acts upon the particles is computed using standard molecular force fields.
-
-#### Example
-
-Initialize the scenario:
-
-```python
-from inductiva import molecules
```
Comment (sarmento) [inline README.md:54]: Keep the example code in the main page (even if it is duplicated elsewhere) as a way of illustrating how scenarios work. It follows naturally from the previous paragraph.

File: README.md [hunk 2]
```diff
@@ -40,43 +40,6 @@ And you are good to go! You can start [exploring our tutorial notebooks](https:/
...
-
-Visualize the results:
-```python
-view = ouptut.render_interactive(representation="ball+stick", add_backbone=True)
-```
-
-This yields an interactive visualization of the protein's trajectory that can be visualized and manipulated in a standard jupyter notebook. The user can specify the representation used for the protein and choose to add the backbone to the visualization.
-
-![Protein solvation simulation.](resources/media/protein_solvation.gif)
-
```
Comment (sarmento) [inline README.md:43]: Had here a section on "Available Scenarios". You can start by listing the Protein Solvation where you point to the "expanded" documentation.

**Review by sarmento — APPROVED** [review]
LGTM!

File: README.md [hunk 3]
```diff
@@ -40,44 +40,25 @@ And you are good to go! You can start [exploring our tutorial notebooks](https:/
...
 
-Visualize the results:
 ```python
-view = ouptut.render_interactive(representation="ball+stick", add_backbone=True)
-```
-
-This yields an interactive visualization of the protein's trajectory that can be visualized and manipulated in a standard jupyter notebook. The user can specify the representation used for the protein and choose to add the backbone to the visualization.
+ import inductiva
 
-![Protein solvation simulation.](resources/media/protein_solvation.gif)
```
Comment (hpenedones) [inline README.md:78]: I think you can leave the image, no?

Comment (cristianacarpinteiro) [reply README.md:78]: ezgif com-video-to-gif
Put this one which is bigger.

**Pull request merged**